src/rpc/RPCServer.ts

```diff
@@ -202,7 +202,7 @@ class RPCServer {
             }
             controller.enqueue(value);
           } catch (e) {
-            if (rpcUtils.isReturnableError(e)) {
+            if (reason == null) {
```

Comment:
What was the reason behind this? That is changing from `isRetunrable` to `null`?

Reply:
After looking over the placeholder error I determined that it wasn't needed and the current method didn't really work. Currently the only time we want to not send an error through the stream is when we can't. So now we're only checking if the stream has already failed. if so we emit the error through the event system otherwise we just send the error message.

Reply:
How come you don't check `e`? You're checking `reason` here.

Reply:
Reason is set by the streams `abort` handler. It lets us know that we can't enqueue the error message if we wanted to. It `reason` could use a more descriptive name in that case.

Reply:
What would `e` be if `reason` is `null`?

Reply:
`e` would be an error thrown in the handler if it failed to do something.

Reply:
> Reason is set by the streams `abort` handler. It lets us know that we can't enqueue the error message if we wanted to. It `reason` could use a more descriptive name in that case.

You mean the `cancel` handler?

Reply:
See discussion https://github.com/MatrixAI/Polykey/pull/509#issuecomment-1480552627